### ISSUE TITLE
При создании элемента формы сделать автоматическую смену заголовка.

### ISSUE THREAD

**Issue by lrRatibor** [1C-Company/1c-edt-issues#1062]
### Описание проблемы

Сейчас, при создании элемента формы, после ввода имени, заголовок остается не измененным.

### Описание решения проблемы

![image](https://github.com/1C-Company/1c-edt-issues/assets/125652316/c9e3bd9f-b4ff-438b-8851-96427a5bae10)

Слева, как происходить сейчас. Справа, как хотелось бы видеть.

**Comment by pns007** [comment]
очень удобно в конфигураторе. Совсем не удобно в edt повторно набирать. Тратиться время.

**Comment by adminimusRU** [comment]
При изменении имени уже созданного элемента в панели свойств заголовок обновляется как и в конфигураторе. Видимо в диалоге создания просто забыли реализовать.

**Comment by lrRatibor** [comment]
> При изменении имени уже созданного элемента в панели свойств заголовок обновляется как и в конфигураторе. Видимо в диалоге создания просто забыли реализовать.

да, для редактирования ранее созданных всё ОК, а именно с новыми не совсем удобно.
